contracts/contracts/rem.system/src/rem.system.cpp

```diff
@@ -71,6 +80,23 @@ namespace eosiosystem {
       _global2.set( _gstate2, _self );
       _global3.set( _gstate3, _self );
       _global4.set( _gstate4, _self );
+      _global5.set( _gstate5, _self );
+   }
+
+   void system_contract::setstakeshare( double share ) {
```

Comment:
let's combine into one method that sets stake+vote+savings, so we can add a check that total share equals 1